src/content/docs/release-notes/docs-release-notes/docs-10-25-2024.mdx

```diff
@@ -0,0 +1,65 @@
+---
+subject: "Docs"
+releaseDate: '2024-10-25'
+version: 'version: October 18-24, 2024'
+---
+
+### New docs
+
+* New troubleshooting doc [PHP agent overhead reduction tips](/docs/apm/agents/php-agent/troubleshooting/php-agent-overhead-reduction-tips) lists configuration settings you can change to reduce the agent's overhead.
+
+### Major changes
+
+* We've released our new Advanced Compute add-on and updated various pricing, billing, user, and product license docs to reflect this.
+* We've revamped our Docs site search to give you a separate search results page. Check it out and let us know what you think!
+* Updated our Kubernetes agent operator doc to include information on [Kubernetes APM auto-attach](/docs/kubernetes-pixie/kubernetes-integration/installation/k8s-agent-operator).
+
+### Minor changes
+
+* When [testing the Python agent](/docs/apm/agents/python-agent/troubleshooting/testing-python-agent/#host-name-lookup) to troubleshoot collector issues, updated outdated hostname lookup IP addresses.
+* In [instrument your non-containerized Lambda function](/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/instrument-your-own/#cdk), added example code for a basic CDK app that deploys a Node.js Lambda function instrumented by New Relic.
```

Comment:
Suggestion: In [instrument your non-containerized Lambda function](/docs/serverless-function-monitoring/aws-lambda-monitoring/instrument-lambda-function/instrument-your-own/#cdk), added example code for a basic CDK app that deploys a Node.js.

We can remove redundant trailing words like "Lambda function instrumented by New Relic".

Reply:
Absolutely right!

Reply:
I do think we need to mention the Lambda function there, even if it's a bit redundant with the doc title